lib/AnsiSerializer.js

```diff
@@ -14,6 +14,7 @@ Object.keys(ansiStyles).forEach(function (styleName) {
 
 function AnsiSerializer(theme) {
     this.theme = theme;
+    this.supportsColor = require('supports-color') || {};
```

Comment:
Wouldn't just be a matter of using the following test below: `supportsColor && supportsColor.has256`?